specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -11927,6 +11969,37 @@
           "$ref": "#/definitions/AzureAvsNodeType",
           "description": "AVS node type."
         },
+        "nodeTypes": {
+          "type": "array",
+          "description": "AVS node types.",
+          "items": {
+            "$ref": "#/definitions/AzureAvsNodeType"
+          }
+        },
+        "externalStorageTypes": {
+          "type": "array",
+          "description": "AVS external storage types.",
```

Comment:
For Array, description "List of AVS external storage types" looks better.

Reply:
sure. will update the description